lib/myapp_web/live/product_live/form_component.ex

```diff
@@ -34,7 +33,7 @@ defmodule MyappWeb.ProductLive.FormComponent do
       result =
         HTTPoison.get('http://slowwly.robertomurray.co.uk/delay/250/url/http://www.google.co.uk')
 
-      {:result, socket.assigns.myself, result}
+      {:result, socket.assigns.id, result}
```

Comment:
Turns out `myself` is supposed to just be an internal identifier, it isn't the `id` used within the parent to track the live component identity. It only _happened_ to work last time by accident. Using `@myself` should probably _never_ work, I'll be filing a bug on live view.